tests/test_init.py

```diff
@@ -53,3 +57,54 @@ def test_parse_bad_patchwork_url(self):
             "Can't parse patchwork url: '{}'".format(patchwork_url)
             in context.exception
         )
+
+    def test_ktree_init(self):
+        """Test ktree.__init__ without specifying a work directory"""
```

Comment:
This doesn't say what exactly is being tested. Did you mean "Test ktree can be created without specifying a work directory"?